ext/openssl/openssl.stub.php

```diff
@@ -95,105 +92,100 @@ function openssl_pkey_free(OpenSSLAsymmetricKey $key): void {}
  */
 function openssl_free_key(OpenSSLAsymmetricKey $key): void {}
 
-/** @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $key */
-function openssl_pkey_get_private($key, string $passphrase = UNKNOWN): OpenSSLAsymmetricKey|false {}
+/** @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $private_key */
+function openssl_pkey_get_private($private_key, ?string $passphrase = null): OpenSSLAsymmetricKey|false {}
 
 /**
- * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $key
+ * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $private_key
  * @alias openssl_pkey_get_private
  */
-function openssl_get_privatekey($key, string $passphrase = UNKNOWN): OpenSSLAsymmetricKey|false {}
+function openssl_get_privatekey($private_key, ?string $passphrase = null): OpenSSLAsymmetricKey|false {}
 
 function openssl_pkey_get_details(OpenSSLAsymmetricKey $key): array|false {}
 
-function openssl_pbkdf2(string $password, string $salt, int $key_length, int $iterations, string $digest_algorithm = 'sha1'): string|false {}
+function openssl_pbkdf2(string $passphrase, string $salt, int $key_length, int $iterations, string $digest_algorithm = "sha1"): string|false {}
 
-function openssl_pkcs7_verify(string $filename, int $flags, string $signerscerts = UNKNOWN, array $cainfo = UNKNOWN, string $extracerts = UNKNOWN, string $content = UNKNOWN, string $pk7 = UNKNOWN): bool|int {}
+function openssl_pkcs7_verify(string $filename, int $flags, ?string $output_filename = null, array $ca_info = [], ?string $untrusted_certificates_filename = null, ?string $content = null, ?string $pk7_filename = null): bool|int {}
```

Comment:
If you are already changing that parameter, then pk7 is of course incorrect as it doesn't mean anything. It should be `pkcs7_filename`

Reply:
From the really quick look I'm not 100% sure what's the use case of that param. It's a bit confusing because `output_filename` is not really right name here. It should be probably called `signers_certificates_filename` and the last one should be probably `pkcs7_output_filename`. Although it's a bit inconsistent with other naming.

Reply:
also first param should be then `pkcs7_input_filename` which will make it a bit more consistent.